ci/pipeline.yml

```diff
@@ -273,6 +272,56 @@ jobs:
           FLY_USERNAME: ((concourse-username))
           FLY_PASSWORD:  '((concourse-password))'
 
+  - name: update-living-documentation
```

Comment:
could you please rename it to update-documentation, because we will include another task to update readme TOC